std/asys/uv/UVErrorType.hx

```diff
@@ -1,81 +1,81 @@
-	var EAI_SERVICE = -3010; // "service not available for socket type"
-	var EAI_SOCKTYPE = -3011; // "socket type not supported"
-	var EALREADY = -37; // "connection already in progress"
-	var EBADF = -9; // "bad file descriptor"
-	var EBUSY = -16; // "resource busy or locked"
-	var ECANCELED = -89; // "operation canceled"
-	var ECHARSET = -4080; // "invalid Unicode character"
-	var ECONNABORTED = -53; // "software caused connection abort"
-	var ECONNREFUSED = -61; // "connection refused"
-	var ECONNRESET = -54; // "connection reset by peer"
-	var EDESTADDRREQ = -39; // "destination address required"
-	var EEXIST = -17; // "file already exists"
-	var EFAULT = -14; // "bad address in system call argument"
-	var EFBIG = -27; // "file too large"
-	var EHOSTUNREACH = -65; // "host is unreachable"
-	var EINTR = -4; // "interrupted system call"
-	var EINVAL = -22; // "invalid argument"
-	var EIO = -5; // "i/o error"
-	var EISCONN = -56; // "socket is already connected"
-	var EISDIR = -21; // "illegal operation on a directory"
-	var ELOOP = -62; // "too many symbolic links encountered"
-	var EMFILE = -24; // "too many open files"
-	var EMSGSIZE = -40; // "message too long"
-	var ENAMETOOLONG = -63; // "name too long"
-	var ENETDOWN = -50; // "network is down"
-	var ENETUNREACH = -51; // "network is unreachable"
-	var ENFILE = -23; // "file table overflow"
-	var ENOBUFS = -55; // "no buffer space available"
-	var ENODEV = -19; // "no such device"
-	var ENOENT = -2; // "no such file or directory"
-	var ENOMEM = -12; // "not enough memory"
-	var ENONET = -4056; // "machine is not on the network"
-	var ENOPROTOOPT = -42; // "protocol not available"
-	var ENOSPC = -28; // "no space left on device"
-	var ENOSYS = -78; // "function not implemented"
-	var ENOTCONN = -57; // "socket is not connected"
-	var ENOTDIR = -20; // "not a directory"
-	var ENOTEMPTY = -66; // "directory not empty"
-	var ENOTSOCK = -38; // "socket operation on non-socket"
-	var ENOTSUP = -45; // "operation not supported on socket"
-	var EPERM = -1; // "operation not permitted"
-	var EPIPE = -32; // "broken pipe"
-	var EPROTO = -100; // "protocol error"
-	var EPROTONOSUPPORT = -43; // "protocol not supported"
-	var EPROTOTYPE = -41; // "protocol wrong type for socket"
-	var ERANGE = -34; // "result too large"
-	var EROFS = -30; // "read-only file system"
-	var ESHUTDOWN = -58; // "cannot send after transport endpoint shutdown"
-	var ESPIPE = -29; // "invalid seek"
-	var ESRCH = -3; // "no such process"
-	var ETIMEDOUT = -60; // "connection timed out"
-	var ETXTBSY = -26; // "text file is busy"
-	var EXDEV = -18; // "cross-device link not permitted"
-	var UNKNOWN = -4094; // "unknown error"
-	var EOF = -4095; // "end of file"
-	var ENXIO = -6; // "no such device or address"
-	var EMLINK = -31; // "too many links"
-	var EHOSTDOWN = -64; // "host is down"
-	var EOTHER = 0;
+extern enum abstract UVErrorType(Int) {
+	var E2BIG; // "argument list too long"
```

Comment:
Why not turn thse line comments into doc comments? Otherwise, the info won't show up in IDE completion or dox.

Reply:
Will do. The same strings are also used when stringifying in `haxe.Error`, but docstrings should be available too. I think I wanted a longer, more meaningful description for the errors, but that can be done later anyway.